01_prepare_host.sh

```diff
@@ -43,6 +43,14 @@ if ! command -v kustomize 2>/dev/null ; then
     sudo mv kustomize /usr/local/bin/.
 fi
 
+if [ "${EPHEMERAL_CLUSTER}" == "kind" ]; then
+    if ! command -v kind 2>/dev/null ; then
+        curl -Lo ./kind https://github.com/kubernetes-sigs/kind/releases/download/"${KIND_VERSION}"/kind-"$(uname)"-amd64
+        chmod +x ./kind
+        sudo mv kind /usr/local/bin/.
+    fi
+fi
```

Comment:
The download and setup of minikube should also be parameterized, to happen only if the ephemeral cluster is not kind or is minikube.

Reply:
Fixed